src/components/external/PaymentLinkDetails/components/PaymentLinkDetails/PaymentLinkDetails.tsx

```diff
@@ -74,11 +115,11 @@ export const PaymentLinkDetails = ({ id, onUpdate, hideTitle }: ExternalUICompon
         setActiveScreen('expirationConfirmation');
     };
 
-    const handleExpirationSuccess = () => {
+    const handleExpirationSuccess = useCallback(() => {
         setActiveScreen('details');
```

Comment:
Should we show a success screen when the link is expired? Now it's going directly to details, we might be a bit confusing for the users as they don't get an immediate feedback that the expiration was successful